examples/EvaluationDataToApplicationInsights/Pages/Checkout.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿@page
+@model CheckoutModel
+@{
+    ViewData["Title"] = "Checkout";
+}
+<h1>@ViewData["Title"]</h1>
+
+<p>Click Below To Check Out!</p>
+
+<button id="checkout" name="checkout" onclick="appInsights.trackEvent({ name: 'checkout', properties: { 'success' : 'yes' } }); appInsights.trackMetric({ name: 'checkoutAmount', average: Math.floor(Math.random() * 100) }); alert('Checked Out!');">
```

Comment:
Also worth calling out the events/metrics you have wired up in the README. This are of great interest when running the example. Perhaps even how to query them.

Reply:
Good call. Added a snippet about it.